packages/blunderbuss/test/fixtures/config/pr_on_always.yml

```diff
@@ -0,0 +1,9 @@
+assign_prs_by:
+- labels:
+  - 'samples'
+  to:
+  - java-samples-reviewers
+assign_prs:
+- prs1
+assign_prs_always:
```

Comment:
Rather than introducing an additional `assign_prs_always` which seems a little redundant given that we have `assign_prs`, what if you instead introduce:

```
assignment_config:
  apply_all_pr_rules: true
```

Or something like this.

Reply:
If `apply_all_pr_rules` was set to `true`, it implies we should apply both `assign_prs_by` and `assign_prs`.

Reply:
I prefer @amanda-tarafa's proposal.  What does @amanda-tarafa say?

Reply:
I also prefer my own proposal (haha) because it means that assign_prs can act as a fallback for assign_prs_by regardless of whether assign_prs_always is set. For instance, for products whose samples don't have an owner (i.e. we don't have an assign_prs_by), we still need two reviews, so we still need two assignees. We would get one from assign_prs (as a fallback for assign_prs_by) and one from assign_prs_always.

Reply:
I prefer my proposal, but am outvoted 😝 . Approved!